<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find reflex_control)/config/reflex_control.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  
  args="--stopped joint_impedance_controller cartesian_impedance_controller"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find reflex_control)/launch/robot.rviz"/>
</launch>
